### PR TITLE
fix(deps): update pjs api

### PR DESCRIPTION
**Pull request by jsdw**
Bump pjs API to 8.8.2 and a wee tidy in the release docs.

### CODE REVIEW

**Comment by TarikGul** [comment]
So to fix the issue here we need to update the `ISanitizedEvent` interface type. It can be found at `src/types/responses/SanitizedEvent.ts`

The file should look like the following:
```
import { Codec } from '@polkadot/types-codec/types';
import { IEventData } from '@polkadot/types/types';

import { IFrameMethod } from '.';

export interface ISanitizedEvent {
	method: string | IFrameMethod;
	data: Codec[] & IEventData;
	docs?: string
}
```

File: .gitignore [hunk 1]
```diff
@@ -31,8 +31,4 @@ yarn-error.log
 *.sublime-workspace
 
 # IDE - VSCode
-.vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+.vscode/*
```
Comment (TarikGul) [inline .gitignore:34]:
```suggestion
.vscode/*

```

**Review by TarikGul — APPROVED** [review]
LGTM, thanks for the updates to the README.md

**Review by lexnv — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -299,18 +299,26 @@ All the commits in this repo follow the [Conventional Commits spec](https://www.
...
+and read the release notes for any breaking changes or high priority updates. In order to update all the dependencies and resolutions run `yarn update-pjs-deps`.
 
     - @polkadot/api [release notes](https://github.com/polkadot-js/api/releases)
     - @polkadot/apps-config [release notes](https://github.com/polkadot-js/apps/releases)
       - If there are any major changes to this package that includes third party type packages, its worth noting to contact the maintainers of sidecar and do a peer review of the changes in apps-config, and make sure no bugs will be inherited.
     - @polkadot/util-crypto [release notes](https://github.com/polkadot-js/common/releases)
     - @substrate/calc [npm release page](https://www.npmjs.com/package/@substrate/calc)
 
-1. Ensure everything is working by running the following tests, `yarn build`, `yarn lint`, `yarn test`, `yarn test:init-e2e-tests`.
+1. Ensure everything is uptodate and working by running the following:
```
Comment (niklasad1) [inline README.md:311]:
```suggestion
1. Ensure everything is up to date and working by running the following:
```

**Pull request merged**